### ISSUE TITLE
AppStore application

### ISSUE THREAD

**Issue by AvivWn** [kuglee/TermiWatch#3]
Why don’t you publish this application for the App Store?

**Comment by kuglee** [comment]
I wish I could, but I'm pretty sure it violates some App Store rules like hiding the default time widget and and acting as a Watch Face.

**Comment by AvivWn** [comment]
Lol
I don’t think it is illegal to hide the default time widget, because you could do it in your code, meaning that they made it possible, right?
About acting as a Watch Face, I don’t know about that...

**Comment by maximdietz** [comment]
Yes, It'd be worth a shot, it's an awesome app and there are plently of apple watch apps I have that hide the time...

**Comment by AvivWn** [comment]
I think that the app it great, but over time I see that it isn’t convenient to use a third party watch face.
I wish apple will update the Apple Watch face main app to support also third party Watch faces.

**Comment by Lapin** [comment]
> I wish I could, but I'm pretty sure it violates some App Store rules like hiding the default time widget and and acting as a Watch Face.

Hello, sorry for pushing up an old thread but there's an app called clockology does allows you to put on custom watch faces. https://apps.apple.com/us/app/clockology/id1456386228

a Apple Watch strap company called rendrstudios offering free watch faces accompanies their watch strap styles through that app. 

Just assumed you'd might be interested :)
